themes/kiali/layouts/partials/home/hero-section.html

```diff
@@ -1,11 +1,12 @@
   <section id="hero">
-    <h1>Service mesh observability<br />and configuration</h1>
+    <h1>Service mesh management<br />for Istio</h1>
 
     <p>
     <strong>Kiali answers the questions:</strong><br />
-    What microservices are part of my Istio service mesh?<br />
+    Which microservices are part of my service mesh?<br />
     How are they connected?<br />
     How are they performing?<br />
+    How can I configure them?<br />
```

Comment:
I'd like to use a high level word "operate" I think.
But I let you to re-phrase this.

Reply:
How about replacing, "How can I configure them"  with "How can I operate on them"?

Reply:
+1